robot_studio_resources/TExtAxisMain.mod

```diff
@@ -61,25 +61,23 @@ MODULE TExtAxisMain
     !
     !***********************************************************
     PROC main()
-        WHILE TRUE DO
-            
-            ! Register an EGM id.
-            EGMGetId egm_id;
-            
-            ! Setup the EGM communication.
-            EGMSetupUC MU_250__01, egm_id, "default", "EXTAX", \Joint;
-            
-            ! Prepare for an EGM communication session.
-            EGMActJoint egm_id;
-            
+        
+        ! Register an EGM id.

```

Comment:
It looks like this file is not mentioned in any of the docs. Can you please add a little mention of it in the docs and a comment at the top of this file.

It could be something like,

`!This is an example of running an external axis. It can be started in parallel with a robot RAPID module`